n2ws/manifest.json

```diff
@@ -26,9 +26,9 @@
   "assets": {
     "configuration": {},
     "dashboards": {
-	  "N2WSBackup&Recovery-EntitiesSpecificDashboard": "assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json",
```

Comment:
It seems assets/dashboards/N2WSBackup&Recovery-EntitiesSpecificDashboard.json has been removed as part of this PR